sdk/src/client/node_manager/mod.rs

```diff
@@ -228,54 +227,39 @@ impl NodeManager {
             for node in nodes {
                 match self.http_client.get(node.clone(), timeout).await {
                     Ok(res) => {
-                        match res.status() {
-                            200 => {
```

Comment:
Will this not return a serialization error now instead of a better node one?

Reply:
naw those are already checked at the http.get().parse:response().
Only thing here could be into_json() failing, turning it into Error::Node(rewest), i guess you mean that. What would you want to do with that?

otherwise, `error.replace(err.into());` the err will be Error::node()